tcms/static/js/testrun_actions.js

```diff
@@ -544,17 +544,9 @@ function changeCaseRunAssignee() {
     return false;
   }
 
-  jQ.ajax({
-    'url': '/runs/update-assignee/',
-    'type': 'POST',
-    'data': { ids: runs, assignee: p },
-    'success': function (data, textStatus, jqXHR) {
-      window.location.reload();
```

Comment:
you have removed this, and this will change behavior. we must find a way to replicate this

Reply:
the way I see it there are 2 options.
option 1:
```
runs.forEach(runId => jsonRPC(....., sync=true))
window.location.reload()
```
this will wait for the runs to be updated and then refresh the page. drawback - it will be if you have many runs

option 2:
```
runs.forEach(runId => jsonRPC(....., sync=false))
window.location.reload()
```
this will dispatch the calls to the backend and refresh the page without waiting for them to be completed. this will be faster, but I'm not sure how it will behave when there are many runs.

Reply:
Option 2 will not work reliably. Whenever the page refreshes some requests don't seem to get through, I don't really know why. Bulk operations should always use sync=true.